cmd/cstor-pool-mgmt/app/command/start.go

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
"crd" word is appearing in the cli text and also in variable names, package names etc., Replica crd with the specific customer resource names like - CStrorPool or CStorVolumeReplica etc, when the comment is about watching the objects. 

Also can you add unit tests for this file, that will check for messages displayed when the controllers failed to start. For instance, will the controller fail to start if the CStorPool or CStorVolumeReplica CRDs are not yet loaded into the kubernetes cluster? Will there be failures when this pod is unable to connect with kube-master?

Reply:
@kmova. will change and implement the feature with unit test. Do u mean failure caused due to CRD yaml not applied?

Reply:
As discussed, this scenario is handled.